ocs_ci/utility/ibmcloud.py

```diff
@@ -337,6 +338,56 @@ class IBMCloud(object):
     Wrapper for Ibm Cloud
     """
 
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
+        
+        Raises:
+            ValueError if the list of nodes is empty
```

Comment:
```suggestion
            ValueError: if the list of nodes is empty

```
one empty line after this section, updated in suggestion and colon after the exception